### ISSUE TITLE
Can't change music 

### ISSUE THREAD

**Issue by Taggehh** [nasso/urmusic#2]
We can't change the music after playing one : we must restart the site
Thank you for reading (and for fixing <; )

**Comment by Nasso** [comment]
Yeah, I don't know what causes this, I didn't even changed the code! I'll try to figure out anyway.

**Comment by nasso** [comment]
Closing as it seems to be fixed.

**Comment by nasso** [comment]
Bug is still here, able to reproduce on Firefox, in Debian 9.
Reopening.
